tikv/deploy-tikv-using-ansible.md

```diff
@@ -0,0 +1,338 @@
+
+[TiDB-Ansible](https://github.com/pingcap/tidb-ansible) is a TiDB cluster deployment tool developed by PingCAP, based on Ansible playbook. TiDB-Ansible enables you to quickly deploy a new TiKV cluster which includes PD, TiKV, and the cluster monitoring modules.
+
+## Prerequisites
+
+1. Several target machines that meet the following requirements:
+
+    - 4 or more machines
+
+        A standard TiKV cluster contains 6 machines. You can use 4 machines for testing.
+
+    - CentOS 7.3 (64 bit) or later with Python 2.7 installed, x86_64 architecture (AMD64), ext4 filesystem
+
+        Use ext4 filesystem for your data disks. Mount ext4 filesystem with the `nodelalloc` mount option. See [Mount the data disk ext4 filesystem with options](../op-guide/ansible-deployment#mount-the-data-disk-ext4-filesystem-with-options).
+
+    - Network between machines
+    
+    - Same time and time zone for all machines with the NTP service on to synchronize the correct time.
+    
+        See [How to check whether the NTP service is normal](../op-guide/ansible-deployment#how-to-check-whether-the-ntp-service-is-normal).
+
+    - Create a normal `tidb` user account as the user who runs the service. 
+    
+        The `tidb` user can sudo to the root user without a password. See [How to configure SSH mutual trust and sudo without password](../op-guide/ansible-deployment#how-to-configure-ssh-mutual-trust-and-sudo-without-password).
+    
+    > **Note:** When you deploy TiKV using Ansible, use SSD disks for the data directory of TiKV and PD nodes.
+
+2. A Control Machine that meets the following requirements:
+
+    > **Note:** The Control Machine can be one of the target machines.
+    
+    - CentOS 7.3 (64 bit) or later with Python 2.7 installed
+    - Access to the Internet
+    - Git installed
+    - SSH Mutual Trust configured
+
+        In the Control Machine, you can log in to the deployment target machine using `tidb` user account without a password. See [How to configure SSH mutual trust and sudo without password](../op-guide/ansible-deployment#how-to-configure-ssh-mutual-trust-and-sudo-without-password).
+
+## Step 1: Download TiDB-Ansible to the Control Machine
+
+1. Log in to the Control Machine using the `tidb` user account and enter the `/home/tidb` directory.
+
+2. Download the corresponding TiDB-Ansible version. The default folder name is `tidb-ansible`.
+
+    - Download the 2.0 GA version:
+
+        ```bash
+        git clone -b release-2.0 https://github.com/pingcap/tidb-ansible.git
+        ```
+    
+    - Download the master version:
+
+        ```bash
+        git clone https://github.com/pingcap/tidb-ansible.git
+        ```
+
+    You can turn to the official team (info@pingcap.com) for advice on which version to choose.
```

Comment:
Please change this to:
If you have questions regarding which version to use, email to info@pingcap.com for more information.